client/a8c-for-agencies/sections/overview/sidebar/contact-support/index.tsx

```diff
@@ -7,16 +7,31 @@ import { recordTracksEvent } from 'calypso/state/analytics/actions';
 
 import './style.scss';
 
+export const CONTACT_URL_HASH_FRAGMENT = '#contact-support';
+
 export default function OverviewSidebarContactSupport() {
 	const translate = useTranslate();
 
-	const [ showUserSupportForm, setShowUserSupportForm ] = useState( false );
+	const [ showUserSupportForm, setShowUserSupportForm ] = useState(
+		window.location.hash === CONTACT_URL_HASH_FRAGMENT
+	);
 	const dispatch = useDispatch();
 
-	const onContactSupport = useCallback( () => {
-		dispatch( recordTracksEvent( 'calypso_a4a_overview_contact_support_click' ) );
-		setShowUserSupportForm( true );
-	}, [ dispatch ] );
+	const toggleContactForm = () => {
+		setShowUserSupportForm( ( prevState ) => {
+			const nextState = ! prevState;
+
+			const hashFragment = nextState ? CONTACT_URL_HASH_FRAGMENT : '';
```

Comment:
We could rename this var to contactUrlHashFragment, what do you think?

Reply:
All our variable contants take that 'screaming' snake case format - those that refer to strings anyway, from what I've seen (eg `wp-calypso/client/a8c-for-agencies/sections/sites/constants.ts`, `wp-calypso/client/a8c-for-agencies/sections/sites/features/features.ts`). 
It looks like the naming conventions here support that: `wp-calypso/docs/coding-guidelines/javascript.md` - for 'Variables intended to be used as a [constant]'. 

The idea is that that const can be exported and used elsewhere if needed too - perhaps we want to add the link somewhere else, so it's not necessarily just limited to that file.